donate/settings/base.py

```diff
@@ -0,0 +1,385 @@
+        'wagtail_localize.translation_memory',
+        'wagtail_localize_pontoon',
+
+        'wagtail.contrib.settings',
+        'wagtail.embeds',
+        'wagtail.sites',
+        'wagtail.users',
+        'wagtail.snippets',
+        'wagtail.documents',
+        'wagtail.images',
+        'wagtail.search',
+        'wagtail.admin',
+        'wagtail.core',
+
+        'modelcluster',
+        'taggit',
+        'storages',
+        'django_rq',
+        'django_countries',
+
+        'django.contrib.admin',
+        'django.contrib.auth',
+        'django.contrib.contenttypes',
+        'django.contrib.sessions',
+        'django.contrib.messages',
+        'django.contrib.staticfiles',
+        'django.contrib.sitemaps',
+    ]
+
+    @property
+    def MIDDLEWARE(self):
+        return list(filter(None, [
+            'donate.utility.middleware.TargetDomainRedirectMiddleware'
+            if self.DOMAIN_REDIRECT_MIDDLEWARE_ENABLED else None,
+            'django.middleware.gzip.GZipMiddleware',
+            'django.middleware.security.SecurityMiddleware',
+            'whitenoise.middleware.WhiteNoiseMiddleware',
+            'django.contrib.sessions.middleware.SessionMiddleware',
+            'django.middleware.locale.LocaleMiddleware',
+            'django.middleware.common.CommonMiddleware',
+            'django.middleware.csrf.CsrfViewMiddleware',
+            'django.contrib.auth.middleware.AuthenticationMiddleware',
+            'django.contrib.messages.middleware.MessageMiddleware',
+            'django.middleware.clickjacking.XFrameOptionsMiddleware',
+            'wagtail.core.middleware.SiteMiddleware',
+            'csp.middleware.CSPMiddleware',
+        ]))
+
+    ROOT_URLCONF = 'donate.urls'
+
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'APP_DIRS': True,
+            'DIRS': [
+                app('templates'),
+            ],
+            'OPTIONS': {
+                'context_processors': [
+                    'django.template.context_processors.debug',
+                    'django.template.context_processors.request',
+                    'django.contrib.auth.context_processors.auth',
+                    'django.contrib.messages.context_processors.messages',
+                    'wagtail.contrib.settings.context_processors.settings',
+                ],
+            },
+        },
+    ]
+
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': root('db.sqlite3'),
+        }
+    }
+
+    RQ_QUEUES = {
+        'default': {
+            'URL': 'redis://redis:6379/0',
+            'DEFAULT_TIMEOUT': 500,
+        },
+
+        # Must be a separate queue as it's limited to one item at a time
+        'wagtail_localize_pontoon.sync': {
+            'URL': 'redis://redis:6379/0',
+            'DEFAULT_TIMEOUT': 500,
+        },
+    }
+
+    LANGUAGES = [
```

Comment:
a nit, but would it be possible to make this a dedicated list in something like `languages.py` and then importing from there?

Reply:
yeah I'm okay with that.